src/main/scala/com/fortysevendeg/translatebubble/modules/clipboard/impl/ClipboardServicesComponentImpl.scala

```diff
@@ -103,6 +103,18 @@ trait ClipboardServicesComponentImpl
 
   }
 
+  private def isValidText(text: CharSequence): Boolean = {
+    !isTrimmedTextEmpty(text) && !isTextANumber(text) && !isTextAUrl(text)
```

Comment:
Looks fine for you if we use something like this?

```
  private def isValidText(text: String): Boolean =
    text.trim.nonEmpty && !isTextANumber(text) && !isTextAUrl(text)
```
Note: braces are not needed here ;)